.github/workflows/swagger-convert.yml

```diff
@@ -0,0 +1,66 @@
+name: Download and commit updated OPENAPI spec files
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  #workflow_dispatch:
+  #schedule:
+  #  - cron:  '0 18 * * *' # Everyday at 6PM (before CI/CD)
+jobs:
```

Comment:
@hbenali I commented those lines to check with you if everything is fine to push ?
Is using the secret ``` ${{ secrets.SWF_TOKEN }} ``` valid for this project ? shouldn't we use ``` ${{ secrets.GITHUB_TOKEN }} ```

Reply:
`GITHUB_TOKEN` refers to the githubot user who is a normal user. When we set protection in the branch (disable direct push) this user will be unable to push commits. `SWF_TOKEN` is an admin generated token with reduced peiveleges scope. It allows to bypass branch protection rule as the push is triggered by a repo admin